github_copilot_seats/seats.py

```diff
@@ -0,0 +1,113 @@
+import json
+import os
+from datetime import datetime, timezone, timedelta
+from typing import Any
+import autokitteh
+from autokitteh.github import github_client
+from autokitteh.slack import slack_client
+from helpers import github_username_to_slack_user_id
+
```

Comment:
Black/Ruff won't complain or fix this automatically, but let's be consistent about 2 empty lines between imports and everything else, just line Black/Ruff **do** automatically enforce 2 empty lines between module-level functions.
```suggestion


```